### DISCUSSION TITLE
Supported Platforms?

### DISCUSSION THREAD

**Discussion by rparys** [resync-tv/resync#54] [Q&A] [answered]
Hello there.

Beautiful and modern alternative to watch2gether.
But which platforms are currently supported and which are still to be supported?

Cheers.

**Comment by vaaski** [comment, marked as answer]
Hey, thank you so much for checking out resync. You seem to be the first one to naturally find and interact with the project!

Right now only YouTube is fully supported and tested for, but you can paste whatever direct stream URL you want.
resync uses [yt-dlp](https://github.com/yt-dlp/yt-dlp) on the backend which [supports a few more sites](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md), so it's basically just a matter of whitelisting and validating.

Do you have a specific site in mind you'd like to see added?
